### ISSUE TITLE
Create standalone Feedback Profile

### ISSUE THREAD

**Issue by andrew-gardener** [IMSGlobal/caliper-spec#323]
Based on [IMSGlobal/caliper-central#194](https://github.com/IMSGlobal/caliper-central/issues/194)

- new FeedbackEvent
- new ReactionAnnoation (subTypeOf Annoation)
- new RatingAnnoation (subTypeOf Annoation)

**Comment by andrew-gardener** [comment]
I heard from my college that the next step for the feedback profile proposition was to create an respec for it. I tried to follow what was done in the search profile branch.

I currently don't have permissions for this repo so I've pushed what I currently have done here: https://github.com/andrew-gardener/caliper-spec/tree/develop-profile/Feedback-cs323

Thanks

**Comment by andrew-gardener** [comment]
I've updated the Respec documentation with some of the feedback provided by the working group from the 24 July call.

- switched to using a `Feedback` entity instead of annotations
- added `Scale` and `LikertScale` entities (needs refining)
- switched `FeedbackEvent`'s object to be an `Entity` instead of `DigitalResource` (didn't realize that `Response`'s super type was Entity so I changed it to be as unconstrained as possible).

Is there anything that I'm missing for that branch to be created so I can make the PR?

Thanks
